apps/web/src/services/iam/components/UserGroupManagementTabUsers.vue

```diff
@@ -54,7 +85,7 @@ const handleSelect = async (index) => {
                 <p-heading heading-type="sub"
                            use-selected-count
                            use-total-count
-                           :total-count="userGroupUsersPageState.totalCount"
+                           :total-count="0"
```

Comment:
👀: Is it okay not to apply the data?

Reply:
🙏🏻 I updated it at [commit](https://github.com/cloudforet-io/console/pull/5206/commits/ba77fd640c30eab527cffdb00fe2a4b15f0d966a)  
-  `state.userItemTotalCount` defined with `computed<number>(() => userGroupPageState.users.totalCount)`.